php.php

```diff
@@ -0,0 +1,18 @@
+<?php
+    class Deposit {
+
+        /**
+         * the deposit ownner.
+         *
+         * @var string witout info
+         */
+        private $_owner;
+
+        /*
```

Comment:
Typo inside comment in token 'coment'. Possible corrections:cement (63),comment (51),moment (44)